src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -196,9 +196,13 @@ class UserProfileAdd extends Component {
                         value={firstName}
                         onChange={(e) => this.handleUserProfile(e)}
                         placeholder="First Name"
-                        invalid={!!this.state.formErrors.firstName}
+                        invalid={!!(this.state.formSubmitted && this.state.formErrors.firstName)}
                       />
-                      <FormFeedback className={fontWeight}>{this.state.formErrors.firstName}</FormFeedback>
+                       {this.state.formSubmitted && this.state.formErrors.firstName && (
```

Comment:
Hi @yashwanth170 I think there is some duplicate logic in the code. The Reactstrap `<FormFeedback>` element will only show up when the `invalid `attribute on the `<Input>` element is set to `true`. no need for conditionally rendering `<FormFeedback>`